hbw/app/assets/stylesheets/hbw/branding.sass

```diff
@@ -398,3 +398,46 @@ input.hbw-checkbox
         color: var(--hbw-file-list-deleted-link-color, var(--base-hbw-file-list-deleted-link-color))
   a
     color: var(--hbw-file-list-link-color, var(--base-hbw-file-list-link-color))
+
+.hbw-body, #hbw-bp-control-buttons-container
+  input.hbw-checkbox
+    &[type=checkbox]
+      display: none
+
+  input.hbw-checkbox
+    &[type=checkbox]
+      + span:before
+        font-family: "Font Awesome 5 Free"
+        display: inline-block
+        color: var(--hbw-checkbox-symbol-color, var(--base-hbw-checkbox-symbol-color))
+
+  input.hbw-checkbox
+    &[type=checkbox]:checked
+      + span:before
+        content: var(--hbw-checked-checkbox-symbol, var(--base-hbw-checked-checkbox-symbol))
+
+  input.hbw-checkbox
+    &[type=checkbox]
+      + span:before
+        content: var(--hbw-checkbox-symbol, var(--base-hbw-checkbox-symbol))
```

Comment:
DRY it like
```
input.hbw-checkbox[type=checkbox]
  ...
  & + span:before
    ...
```
and so on, please

Reply:
Also, look at https://github.com/latera/homs/blob/64ca90efab495a17cb64e1faa555c1d8635ea3cd/hbw/app/assets/stylesheets/hbw/branding.sass#L374-L377
looks like it should be placed here too.